packages/ui-extensions/docs/surfaces/point-of-sale/reference/examples/targets/pos-draft-order-details-action-menu-item.ts

```diff
@@ -1,13 +1,16 @@
-import {ActionItem, extension} from '@shopify/ui-extensions/point-of-sale';
+import {Button, extension} from '@shopify/ui-extensions/point-of-sale';
 
 export default extension(
   'pos.draft-order-details.action.menu-item.render',
   (root, api) => {
-    const actionItem = root.createComponent(ActionItem, {
-      onPress: () => api.action.presentModal(),
-      enabled: true,
+    const button = root.createComponent(Button, {
+      title: 'Press me!',
```

Comment:
We should remove title from all the examples in the menu-item cases, since we don't actually render it. We instead force the `description` field to be rendered here. Can we also update the Button docs for `title` to mention this?

Reply:
For sure! Does this mean that the `title` prop should be changed to optional?

![image](https://github.com/user-attachments/assets/4c564643-e34a-496d-b1bd-19314b568c13)

Reply:
Hah, that's a good question. Title is required but ignored by POS in this instance. Yes, I would say making it optional is a good call.

Reply:
Thanks @nate, I've updated the Button's title prop to Optional and added a note for clarity.
![image](https://github.com/user-attachments/assets/7c25fd46-d7e0-4cae-a176-2947dd82ec04)